### PR TITLE
Create notebook introducing data collections

### PR DESCRIPTION
**Pull request by emmarogge**
This notebook contains widgets to assist users in creating a workspace then converting it to a data collection. It also provides instructions and an embedded screencast of how to add the data collection to a Terra workspace.

### CODE REVIEW

File: first_hour_on_terra/README.md
Comment (wnojopra) [inline first_hour_on_terra/README.md]: Fix the todo link

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb]: I tend to prefer notebook cells to output **something**, just to show that it actually did something. Here you could add something like `print(f"Current workspace ID is {CURRENT_WORKSPACE_ID}")`

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb]: I think it'd be worth also mentioning here what a data collection is. Pulling directly from our docs site would be fine.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb:161]: Running this cell I get an import error. I think you're missing `from ipywidgets import widgets`?

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb]: As an alternative option, we could add a cell here with `!terra resource list` that one could run after adding the data collection.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb:364]: Cool!

**Review by wnojopra — CHANGES_REQUESTED** [review]
Looks great! A few small things to fix

File: first_hour_on_terra/README.md
Comment (emmarogge) [reply first_hour_on_terra/README.md]: Yes, once this PR merged in to the main branch of this repo, I will be able to replace that link with the actual URL.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: Since I cannot capture that step in the screencast, I'm a little worried it might be awkward to switch back to Bash commands at this point...what do you think? I am happy to include it but not sure where to put it relative to the screencast.

Comment (wnojopra) [reply first_hour_on_terra/working_with_data_collections.ipynb]: I'd also be ok if it said "On a cloud environment terminal in your workspace or in a Jupyter cell". Its a minor point, I don't think it blocks this from being merged.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: fixed!

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: agreed -- added content & links to documenation!

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb:161]: fixed!

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: I have added the command as you suggested -- I tested it out and I am no longer concerned it will be awkward after seeing it.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [reply first_hour_on_terra/working_with_data_collections.ipynb:146]: Instead of a new cell, could you please merge this with the other ipywdigets imports in the first cell?

Comment (wnojopra) [reply first_hour_on_terra/working_with_data_collections.ipynb:161]: Can you double check your imports? For example I see you're using `Layout` and `os` without importing them, which causes import errors. What I'd suggest doing is clicking on kernel -> restart kernel and then running the notebook again from the top.

File: first_hour_on_terra/README.md
Comment (wnojopra) [reply first_hour_on_terra/README.md]: We can figure out that the URL will be https://github.com/DataBiosphere/terra-axon-examples/blob/main/first_hour_on_terra/working_with_data_collections.ipynb . So you can put that in if you wanted to do sooner than later.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [reply first_hour_on_terra/working_with_data_collections.ipynb]: Your instructions said `terra resource list` but your new command says `!terra workspace list`. Which should it be? And could you please also add some markdown briefly mentioning what they should be seeing? Since the cell is divided from the instructions by the video the step "Run the cell below" can be missed.

Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: Per discussion over chat, I've removed this step.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb:161]: I have refactored this and cleaned up the unnecessary imports.

File: first_hour_on_terra/README.md
Comment (emmarogge) [reply first_hour_on_terra/README.md]: updated!

File: first_hour_on_terra/README.md
Comment (wnojopra) [inline first_hour_on_terra/README.md]: Nit: Cleanup the stray space at the start of the URL

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb]: I think it needs to be more clear what the user is expected to input here. Please add instructions here that tells the user that they're expected to create a new workspace id, description, and resource name, and give examples of good names. You have good examples in the expected output, you can reuse those in the instructions.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb]: Nit: Fix the typo with 0Workspace

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb:235]: Same comment as above about making it clear what the user is expected to input. Please add instructions here that tells the user that they're expected to input the SAME workspace id, a description (is this expected to be same or different?), and what version is expected to be. And again please give examples of good names.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb:191]: What happens if this command fails? Say, I give a workspace ID that already exists.

**Review by wnojopra — CHANGES_REQUESTED** [review]

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb:235]: updated!

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: fixed!

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: updated!

File: first_hour_on_terra/README.md
Comment (emmarogge) [reply first_hour_on_terra/README.md]: fixed!

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb:191]: I have now updated both widgets to print the success message (from stdout) if the command succeeds, but to otherwise print the error message (from stderr) so that users can learn from the error message what went wrong.

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (wnojopra) [inline first_hour_on_terra/working_with_data_collections.ipynb]: The workspace_id HAS to match, right? "Must match" instead of Should match.

**Review by wnojopra — APPROVED** [review]
Awesome! One more comment but otherwise LGTM

File: first_hour_on_terra/working_with_data_collections.ipynb
Comment (emmarogge) [reply first_hour_on_terra/working_with_data_collections.ipynb]: Fixed!

**Pull request merged**
